java/core/src/main/java/com/microsoft/bond/BondEnum.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.bond;
+
+/**
+ * Denotes a Bond enumeration type.
+ * All generated Bond enum classes implement this interface.
+ */
+public interface BondEnum {
```

Comment:
OK

---
In reply to: [122853157](https://github.com/Microsoft/bond/pull/502#discussion_r122853157) [](ancestors = 122853157)